documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -3505,6 +3505,31 @@ In these abnormal situations, {prodname}, like Kafka, provides _at least once_ d
 
 The rest of this section describes how {prodname} handles various kinds of faults and problems.
 
+[id="oracle-logs-do-not-contain-offset-perform-new-snapshot"]
+=== Logs do not contain offset, perform a new snapshot
+
+When the {prodname} Oracle connector restarts, you may find that the connector reports the following error:
+
+[source]
+----
+Online REDO LOG files or archive logs do not contain the offset scn xxxxxxx. Please perform a new snapshot.
```

Comment:
For the MySQL connector we have the [`when_needed`](https://debezium.io/documentation/reference/stable/connectors/mysql.html#mysql-property-snapshot-mode) snapshot mode; looks like that'd be a worthwhile addition for the Oracle connector, too? If you agree, @Naros, can you log an issue for this in Jira?

Reply:
Done, logged [DBZ-4612](https://issues.redhat.com/browse/DBZ-4612).